src/components/app.test.js

```diff
@@ -8,32 +8,41 @@ it('renders without crashing', () => {
   shallow(<App />);
 });
 
-it('displays default content', () => {
-  const div = document.createElement('div');
-  render((
-    <MemoryRouter>
-      <App />
-    </MemoryRouter>
-  ), div);
-  console.assert(div.innerHTML.match(/Easily submit and score QPP data in real-time via API/));
-});
+const contentTestStrings = {
+  '/': 'Easily submit and score QPP data in real-time via API',
+  '/developer-preview': 'Developer Preview',
+  '/tutorial': 'API Tutorial',
+  '/submission': 'The Submissions resource represents one year of performance data'
+};
 
-it('displays the developer-preview', () => {
-  const div = document.createElement('div');
-  render((
-    <MemoryRouter initialEntries={[ '/developer-preview' ]}>
-      <App />
-    </MemoryRouter>
-  ), div);
-  console.assert(div.innerHTML.match(/Developer Preview/));
+Object.keys(contentTestStrings).forEach(function(pathname) {
```

Comment:
Ken, you're officially more helpful than stack overflow https://github.com/CMSgov/qpp-submissions-docs/pull/52/commits/8de33f785face8251bf2abe6159b9cd5e3609dcd